### DISCUSSION TITLE
Target SDK version 30 requires a minimum of sifnature scheme v2

### DISCUSSION THREAD

**Discussion by RacheliGolan** [airsdk/Adobe-Runtime-Support#1537] [General]
I have the updated version of the SDK.
I have an old air flash app that I want to update a version of it in the store .
I updated it to the most up-to-date SDK version.
When I upload to the store I get the following error:
Target SDK version 30 requires a minimum of sifnature scheme v2. the APK is not signed with this or a later signature scheme
Can anyone help me?

**Comment by marchbold** [comment]
(I've transferred this issue from the apm repository as it doesn't seem related to apm).
